graph/articles/collections.md

```diff
@@ -316,3 +316,148 @@ Content-Type: application/json
    "value": []
 }
 ```
+
+## 11. Collections of structural types (complex types or primitive types)
+
+Collections of entity types are generally preferable to collections of structual types because collections of structural types must be updated as a single unit, meaning that they are overwritten entirely by new contents, rather than be updated relative to the existing contents. 
+Sometimes, structural collection properties are added to a type and then scenarios are discovered later that require a collection of entity types.
+Take the following model with an entity type `foo` that has a collection of `bar`s:
```

Comment:
These updates aim to address new issues in real-life modeling. Can we use the actual Graph examples to inform our guidelines?

Reply:
Thanks, I found a suitable example from graph